src/mem/MemCtrl.py

```diff
@@ -58,6 +58,8 @@ class MemCtrl(QoSMemCtrl):
     # bus in front of the controller for multiple ports
     port = ResponsePort("This port responds to memory requests")
 
+    dram_cache_size = Param.Int('1024', "DRAM cache size")
+    write_allocate_policy = Param.Int('false', "DRAM cache write allocate policy, can be 'allocate on write' or 'no allocate on write'")
```

Comment:
Think this should be a bool.

Also, line length :) How were you able to commit?!?!?!?!?!?!